### ISSUE TITLE
[ENHANCEMENT] In-game rewards editor

### ISSUE THREAD

**Issue by Flavourized** [teozfrank/UltimateVotes#15]
> In an effort to automate some parts of the server, we want to implement an easier way to multiply vote rewards, such as Double Vote reward weeks where we give 2x the rewards each vote.
> 
> However, with the current setup, there is no way we can change the file automatically, using an in game command. Due to the fact that the UV rewards are already in their own file, i believe this might be a little easier to implement than if the rewards were in config.
> 
> First off, to make sure a feature like this doesnt break other server setups, add an option in rewards.yml that is something like useInGameEditor: true/false. The plugin should read the rewards.yml repeatingrewards section line by line.To make this user friendly, with a command (maybe /uv edit <repeatingRewards/Rewards> [group]). It will first output the the current repeatingreward commands with the group they are assigned to (if there are multiple group and the arg is not specified, return to player to specify the group.
> 
> 
> The most important part of this whole suggestion is the ability to change rewards with an in game command.
> Possibly, /uv edit <repeatingRewards/Rewards> <group> <line#> <new command>. This basically allows to run a script which runs commands through console so we can have event such double voting rewards, which last a week or so.

tl;dr ability to change rewards.yml with in game commands so we can automate things like double vote rewards weeks
